target/linux/ath79/image/generic.mk

```diff
@@ -154,6 +154,23 @@ define Device/buffalo_wzr-hp-ag300h
 endef
 TARGET_DEVICES += buffalo_wzr-hp-ag300h
 
+define Device/buffalo_wzr-hp-g300nh
+  ATH_SOC := ar9132
```

Comment:
This won't work with master, so it tells me that have not even buildtested with recent master.

Ah, just saw that you actually based this on 19.07 ...